python/bmg_polling_data_processor.ipynb

```diff
@@ -0,0 +1,979 @@
+{
```

Comment:
Notebooks are great but very noisy in VCS without some pre-commit hooks such as https://github.com/kynan/nbstripout. I don't think we need the interactive element. Could we just have this as a .py file instead?